views/apps/create.html

```diff
@@ -1,151 +1,121 @@
 {{define "body"}}
 
 <div
-  class="w-full lg:w-8/12 mx-auto bg-white rounded-md shadow px-4 lg:px-12 py-4 lg:py-12 mb-10 dark:bg-surface-02dp"
+  class="w-full max-w-sm mx-auto bg-white rounded-md shadow p-4 lg:px-8 lg:py-6 mb-10 dark:bg-surface-02dp"
 >
-  <h2 class="font-bold text-2xl font-headline mb-2 dark:text-white">{{.Name}}</h2>
+  <h2 class="font-semibold text-xl text-center font-headline mb-6 dark:text-white">Pair with app to finalize</h2>
 
-  <div
-    class="bg-green-100 border-t-4 border-green-500 rounded-md px-4 py-3 shadow-md mb-8 dark:bg-green-900"
-    role="alert"
-  >
-    <div class="flex items-center text-green-700 dark:text-green-400">
-      <div class="py-1">
-        <svg
-          class="fill-current h-6 w-6 mr-4"
-          xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 20 20"
-        >
-          <path
-            d="M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z"
-          ></path>
-        </svg>
-      </div>
-      <div>
-        <p class="font-bold">App connection created</p>
-      </div>
-    </div>
-  </div>
+  <div class="font-medium text-xs text-center mb-6 dark:text-white">Open, paste or scan your connection’s pairing secret in the desired app to finalise the connection.</div>
 
-  {{if not (eq .PairingSecret "")}}
-  <p class="mb-4 dark:text-white">Use the pairing secret to connect to your app.</p>
+  <a href="{{.PairingUri}}" class="w-full inline-flex bg-purple-700 cursor-pointer duration-150 focus:outline-none font-medium hover:bg-purple-900 items-center justify-center px-3 py-2 rounded-md shadow text-white transition mb-4">
+    <svg class="mr-2" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think we should directly embed svgs like this, what was the reason for the change?

Reply:
We can't add a fill to svgs if we use them from the img tag

Reply:
@im-adithya what about a template, like `{{template "copyIcon" .}}` and then move the copy icon to a separate html file?

Reply:
Not very sure about that, it works, but then again we have to mention each and every icon used in that file in [echo_handlers.go](https://github.com/getAlby/nostr-wallet-connect/blob/main/echo_handlers.go#L51-L58) to parse it